src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/StormLcmSpec.groovy

```diff
@@ -107,7 +107,7 @@ class StormLcmSpec extends HealthCheckSpecification {
         TimeUnit.SECONDS.sleep(45) //after deploy topology needs more time to actually begin working
 
         then: "Switches are recognized as being deactivated"
-        Wrappers.wait(WAIT_OFFSET) {
+        Wrappers.wait(180) { //can take up to 3 network dumps
```

Comment:
do we have some `constant` for network dump?

Reply:
added